projects/azure/acs-engine/test-image-builds/build-3.9.yaml

```diff
@@ -1,26 +0,0 @@
-kind: BuildConfig
```

Comment:
If you are removing this, then we also need a PR to remove the 3.9 jobs from acs-engine.

Reply:
Search for all job that use `registry.svc.ci.openshift.org/azure/acs-engine-tests:v3.9` and remove them.